Service/RequestService.php

```diff
@@ -451,6 +453,12 @@ public function requestHandler(array $data, array $configuration): Response
             }
         }
 
+        // Get application configuration in and out for current endpoint/global if this is set on current application.
+        // Note: we might want to do this earlier in this function if we want to use this configuration there...
+        if ($this->session->get('application') !== null) {
+            $applEndpointConfig = $this->getApplEndpointConfig();
```

Comment:
applEndpointConfig sounds funny, maybe appEndpointConfig would be better.

Reply:
I don't think it is better, the name should at least have something in it referring to Application, because we are getting (and later on handling) the configuration of an Application, more so than the configuration of a specific endpoint.

Reply:
i misread, appEndpointConfig is a better name indeed